templates/Web/Pages/Vue.List/src/components/ListItem.vue

```diff
@@ -23,7 +23,13 @@
 export default {
   name: "ListItem",
   props: {
-    listItem: Object
+    listItem: {
+      type: Object,
+      required: true,
+      validator: function(value) {
+        return !(value.text === undefined || value._id === undefined);
```

Comment:
Personally I think it's harder to read than `(value.text !== undefined || ...)`

Reply:
@jsondoo

Reply:
it would be `(value.text !== undefined &&` not `||` which is why I think this looks cleaner. Should we start a poll?

Reply:
I think the `value.text !== undefined && value._id !== undefined` but that's just a personal opinion.

Reply:
yea `value.text !== undefined && value._id !== undefined` is more readable to me too

> deMorgans Law